test/fixtures/shadow-dom-app.js

```diff
@@ -0,0 +1,16 @@
+import { Component } from '../../lib';
+import h from 'virtual-dom/virtual-hyperscript';
+
+var ShadowDomApp = document.registerElement('shadow-dom-app', class extends Component {
+  get config() {
+    return {
+      css: 'color: blue;',
```

Comment:
it seems like the `css` prop is useless unless `useShadowDom` is set - should we have some sort of runtime check that you're not trying to use it when `useShadowDom` is false?